charts/argo-workflows/values.yaml

```diff
@@ -0,0 +1,325 @@
+  #   passwordSecret:
+  #     name: argo-postgres-config
+  #     key: password
+  workflowDefaults: {} # Only valid for 2.7+
+  #  spec:
+  #    ttlStrategy:
+  #      secondsAfterCompletion: 84600
+  # workflowWorkers: 32
+  # podWorkers: 32
+  workflowRestrictions: {} # Only valid for 2.9+
+  #  templateReferencing: Strict|Secure
+  telemetryConfig:
+    enabled: false
+    path: /telemetry
+    port: 8081
+  serviceMonitor:
+    enabled: false
+    additionalLabels: {}
+  serviceAccount: argo
+  # Service account annotations
+  serviceAccountAnnotations: {}
+  name: workflow-controller
+  workflowNamespaces:
+    - default
+  containerRuntimeExecutor: docker
+  instanceID:
+    # `instanceID.enabled` configures the controller to filter workflow submissions
+    # to only those which have a matching instanceID attribute.
+    enabled: false
+    # NOTE: If `instanceID.enabled` is set to `true` then either `instanceID.userReleaseName`
+    # or `instanceID.explicitID` must be defined.
+    # useReleaseName: true
+    # explicitID: unique-argo-controller-identifier
+  logging:
+    level: info
+    globallevel: "0"
+  serviceType: ClusterIP
+  metricsServicePort: 8080
+  metricsServicePortName: metrics
+  telemetryServicePort: 8081
+  telemetryServicePortName: telemetry
+  # Annotations to be applied to the controller Service
+  serviceAnnotations: {}
+  # Optional labels to add to the controller Service
+  serviceLabels: {}
+  # Source ranges to allow access to service from. Only applies to
+  # service type `LoadBalancer`
+  loadBalancerSourceRanges: []
+  resources: {}
+  # The list of environment variable definitions to be added to the controller
+  # manages container verbatim.
+  extraEnv: []
+  # Extra arguments to be added to the controller
+  extraArgs: []
+  replicas: 1
+  pdb:
+    enabled: false
+    # minAvailable: 1
+    # maxUnavailable: 1
+  ## Node selectors and tolerations for server scheduling to nodes with taints
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+  ##
+  nodeSelector:
+    kubernetes.io/os: linux
+  tolerations: []
+  affinity: {}
+  # Leverage a PriorityClass to ensure your pods survive resource shortages
+  # ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+  # PriorityClass: system-cluster-critical
+  priorityClassName: ""
+  # https://argoproj.github.io/argo/links/
+  links: []
+
+# executor controls how the init and wait container should be customized
+executor:
+  image:
+    registry: quay.io
+    repository: argoproj/argoexec
+    tag: v3.0.1
+  resources: {}
+  # Adds environment variables for the executor.
+  env: {}
+  # sets security context for the executor container
+  securityContext: {}
+
+server:
+  enabled: true
+  # only updates base url of resources on client side,
+  # it's expected that a proxy server rewrites the request URL and gets rid of this prefix
+  # https://github.com/argoproj/argo/issues/716#issuecomment-433213190
```

Comment:
Should be updated to https://github.com/argoproj/argo-workflows/issues/716#issuecomment-433213190